### ISSUE TITLE
Logged In But Says I'm not

### ISSUE THREAD

**Issue by Automaton49** [dmstern/likers-blocker#139]
I click on the Likers Blocker button and the tab says "You are not logged in." but I clearly am logged into Twitter. I even click on the "Login" button and it just opens up another twitter window but I was never logged out.

**Comment by dmstern** [comment]
@Automaton49 is this issue fixed? I assume so due to your issue #140 .

**Comment by Automaton49** [comment]
No, I logged out manually and logged back in using the button on the app but it STILL says I'm not logged in even though I am. 

Also, if the block waiting list is over 1K and it can only block like 300 or so at a time before I have to log out is there some indicator?

**Comment by dmstern** [comment]
Yes, there is an indicator: it's the battery icon, which says "x remaining blocks":

![FsTs-e7WYAEPnRU](https://user-images.githubusercontent.com/5219707/229286468-0b4c270a-d8f3-466a-b8ec-97a297fc6479.jpg)
https://twitter.com/LikersBlocker/status/1640697650881150976

I wrote something in the release notes about why there is this limit of 500 blocks at a time: https://github.com/dmstern/likers-blocker/releases/tag/v4.0.0
> Since Elon planned to charge developers for using of the Twitter API, we found a way to ensure the continued existence of LikersBlocker without paying Elon by blocking the users locally in your browser instead of an external web service. The downsite: You can now only block 500 users at once. If you want to block more, you have to logout from Twitter and login again.

So, I assume, we can at least close the other issue #140?

But let's get back to your login display problem: As I understand your issue, the extension says you are not logged-in, then you click "login", a new twitter tab is opened, and then the extension still says you're not logged in, right? But how did you get to the point where you could add over 1k users to the blocking queue if the login did not work?
To make sure, we do not misunderstand each other: "Login" in the context of the extension does not mean you have to log in into Twitter. "You are not logged-in" when you actually are logged-in into Twitter, means that the extension does not yet have the login information. To pass over your active Twitter login to the extension, you can click "login", which in your (Twitter-logged-in) case just establishes the connection between your active Twitter login and the extension.

So is this issue maybe more like a cosmetic request, like The extension should say "not connected to Twitter" or something like that instead of "You are not logged-in"?

If I still misunderstand your issue, could you please tell me which browser you use to reproduce the issue?

**Comment by spukiss** [comment]
![image](https://user-images.githubusercontent.com/72319020/233061528-0dd5d5ed-a9bd-4cd2-befb-ef40f391c311.png)
I am loged in on Twitter.

**Comment by GeorgeSpelvin** [comment]
Same problem here.  I was logged in, added a bunch of Likers to be blocked and it was mowing its way through the list.
Every 500, I'd have to log out and in and that's okay BUT it's now saying that I'm logged out and I've logged out/in a few times now and even used the extension's link twice.

I've also closed and re-opened Chrome and even tried using Chrome's task manager to kill and restart the extension.

No difference.
